demo-deploy/infra/terragrunt-aws-environments/terragrunt.hcl

```diff
@@ -0,0 +1,70 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# TERRAGRUNT CONFIGURATION
+# Terragrunt is a thin wrapper for Terraform that provides extra tools for working with multiple Terraform modules,
```

Comment:
I cann't say i like all the repetative .hcl and .lock.hcl files for terragrunt.
Do we really need them all?